releases/goerli/1.11.0/index.html

```diff
@@ -0,0 +1,20 @@
+<html>
+	<head>
+		<title>Goerli Release 1.11.0.</title>
```

Comment:
extra dot (same for L8, L12, L14)
```suggestion
		<title>Goerli Release 1.11.0</title>
```

Reply:
oops